server/templates/server/help_table.html

```diff
@@ -46,7 +46,7 @@
                 <tr bgcolor="#ff6347">
             {% endif %}
                 <th scope="row">{{ help_request.id }}</th>
-                <td>{{ help_request.first_name }} {{ help_request.last_name }}</td>
+                <td>{{ help_request.full_name }}</td>
```

Comment:
correct, but we should split full name on help_request to be first_name and last_name (like we need on volunteers)